source/helpers/math.ts

```diff
@@ -0,0 +1,31 @@
+export function lerp(min: number, max: number, t: number): number {
```

Comment:
Use real words instead of letters. What is `t` times?

Reply:
Those are the standard names for interp functions. Tried to be more explict in latest, wdyt?